upup/models/cloudup/resources/addons/gcp-cloud-controller.addons.k8s.io/k8s-1.23.yaml.template

```diff
@@ -41,6 +41,17 @@ spec:
         env:
         - name: KUBERNETES_SERVICE_HOST
           value: "127.0.0.1"
+        livenessProbe:
```

Comment:
Config copied from our KCM manifest except port. Should be okay to use the same setting because the health check mechanism is shared among controller managers.